rpc/src/lib.rs

```diff
@@ -1,5 +1,6 @@
 #![allow(clippy::integer_arithmetic)]
 mod cluster_tpu_info;
+pub mod entry_notifier_interface;
```

Comment:
I put this in `solana-rpc` because I thought we could piggyback off the TransactionStatusService (with a new `TransactionStatusMessage` variant) to call `notify_entry()`

Reply:
Might move this in a follow-up, depending on where the Entry notification service ends up.